client/student/router.js

```diff
@@ -25,6 +25,7 @@ const parseParams = ({ params }) => {
 const fallbackRoute = { path: '*', component: NotFound };
 
 const router = new Router({
+  mode: 'history',
```

Comment:
Isn't this supposed to be toggled conditionally according to current env variable value? 😕

Reply:
You mentioned that you want the `history` routing to be the default one. I'll update this.

Reply:
Well default not hardcoded one 🙃

Reply:
It's hard to read you mind, Vlatko.